packages/ui/src/pages/About/AboutPage.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+    Copyright (C) 2024 Red Hat, Inc.
```

Comment:
2025?

Reply:
same as before, will take care in https://github.com/tplevko/kaoto-next/pull/27